components/breadcrumb/breadcrumb.html

```diff
@@ -0,0 +1,10 @@
+<nav aria-label="breadcrumb" class="mzp-c-breadcrumb">
+  <ol class="mzp-c-breadcrumb-list">
+    <li class="mzp-c-breadcrumb-item">
+      <a href="#">Parent page</a>
```

Comment:
Since the content in this page is very... minimal, I opted to just hard-code it here and make a `readme.md` for the page notes instead of creating a `breadcrumbs.config.yml` file.